apps/demos/Demos/Chat/Overview/Angular/app/app.component.css

```diff
@@ -0,0 +1,24 @@
+.demo-container {
```

Comment:
Don't forget add empty `description.md` for the techwriter in `Chat\Overview` folder.

Reply:
cf5e1a484814b0d94cd80150e2561a18d5b8f97c